src/wallet/wallet2.h

```diff
@@ -829,6 +885,33 @@ namespace boost
         return;
       }
       a & x.m_pk_index;
+      if (ver < 8)
+      {
+        initialize_transfer_details(a, x, ver);
+        return;
+      }
+      a & x.m_multisig_info;
```

Comment:
I think the following should be inserted here:

    a & x.m_key_image_partial;